tests/Functional/ContentNewsRenderTest.php

```diff
@@ -70,6 +76,70 @@ public function testNewsRendering(): void {
     $node->save();
     $this->drupalGet($node->toUrl());
 
+    // Assert news contacts.
+    $contacts = $this->assertSession()->elementExists('css', 'div#news-contacts');
+
+    $contact_headers = $contacts->findAll('css', 'h2');
+    $this->assertEquals('Contacts', $contact_headers[0]->getText());
+
+    $contact_name = $contacts->findAll('css', 'h3');
+    $this->assertCount(1, $contact_name);
+    $this->assertEquals('general_contact', $contact_name[0]->getText());
+
+    $contact_body = $contacts->findAll('css', '.ecl-editor');
+    $this->assertCount(1, $contact_body);
+    $this->assertEquals('Body text general_contact', $contact_body[0]->getText());
+
+    $contacts_html = $contacts->getHtml();
+    $field_list_assert = new FieldListAssert();
+    $contact_expected_values = [
+      'items' => [
+        [
+          'label' => 'Organisation',
+          'body' => 'Organisation general_contact',
+        ], [
+          'label' => 'Website',
+          'body' => 'http://www.example.com/website_general_contact',
+        ], [
+          'label' => 'Email',
+          'body' => 'general_contact@example.com',
+        ], [
+          'label' => 'Phone number',
+          'body' => 'Phone number general_contact',
+        ], [
+          'label' => 'Mobile number',
+          'body' => 'Mobile number general_contact',
+        ], [
+          'label' => 'Fax number',
+          'body' => 'Fax number general_contact',
+        ], [
+          'label' => 'Postal address',
+          'body' => 'Address general_contact, 1001 Brussels, Belgium',
+        ], [
+          'label' => 'Office',
+          'body' => 'Office general_contact',
+        ], [
+          'label' => 'Social media',
+          'body' => html_entity_decode('&nbsp;') . 'Social media general_contact',
+        ],
+      ],
+    ];
+    $field_list_assert->assertPattern($contact_expected_values, $contacts_html);
+    $field_list_assert->assertVariant('horizontal', $contacts_html);
+
+    // Assert Press contacts.
+    $press = $contacts->findAll('css', '.ecl-u-border-top.ecl-u-border-bottom.ecl-u-border-color-grey-15.ecl-u-mt-s.ecl-u-pt-l.ecl-u-pb-l');
+    $press_link = $press[0]->findAll('css', 'a');
+    $this->assertCount(1, $press_link);
```

Comment:
Reset the array after asserting that only one element was found. Same for the others.
```suggestion
    $this->assertCount(1, $press_link);
    $press_link = reset($press_ink);
```

Reply:
Updated when the array is used after assertCount, the function is reset(&$array);

Reply:
You misunderstood me. After using findAll() which returns an array you assert that is only one element. If you reset the array as I specified $array = reset($array) you'll get the only element so then you don't have to do $press_link[0]->find....
So just do $press_link = reset($press_link) and then you use $press_link everywhere. The same for other situations. After making sure you have only one element, reset the array to get the element instead of using the array further.